src/jit/rationalize.cpp

```diff
@@ -157,19 +157,20 @@ void Compiler::fgFixupArgTabEntryPtr(GenTreePtr parentCall, GenTreePtr oldArg, G
     }
 }
 
-// Rewrite InitBlk involving SIMD vector into stlcl.var of a SIMD type.
+// Rewrite a non-TYP_STRUCT indirection as GT_IND(GT_LEA(obj.op1)), or as a simple
+// lclVar if possible.
 //
```

Comment:
So ... that comment is somewhat anticipatory - I had made additional changes to use the enregisterable struct types, which also need this transformation. I'll add an additional comment.

---
In reply to: [77282496](https://github.com/dotnet/coreclr/pull/7027#discussion_r77282496) [](ancestors = 77282496)